### PR TITLE
Add Coincafe

### PR DESCRIPTION
**Pull request by ilyakatz**

### CODE REVIEW

File: _data/bitcoin.yml [hunk 1]
```diff
@@ -68,7 +68,13 @@ websites:
       sms: Yes
       software: Yes
       doc: http://blog.coinbase.com/post/25677574019/coinbase-now-offers-two-factor-authentication
-      
+
+    - name: Coinbase
```
Comment (mxxcon) [inline _data/bitcoin.yml]: This is Coincafe, not coinbase. :smiley:

**Comment by mxxcon** [comment]
Your logo is 36x36, not 32x32. :wink:

**Comment by ilyakatz** [comment]
oh sorry, haven’t contributed in a while, so apparently forgot the exact dimensions. coming right up
-- 
Ilya Katz
Sent with Airmail

On September 1, 2014 at 1:45:27 PM, mxxcon (notifications@github.com) wrote:

Logo is 36x36, not 32x32.

**Comment by ilyakatz** [comment]
@mxxcon thanks for careful code review. updated the mistakes

**Comment by mxxcon** [comment]
lgtm :+1:

**Pull request merged**